File: Programming Basics with Python/02 Conditional statements/More exercises/08_Fuel_Tank.py
fuel = input().lower()
liters_in_tank = float(input())
if fuel == 'diesel' or fuel == 'gasoline' or fuel == 'gas':
    if liters_in_tank >= 25:
        print(f'You have enough {fuel}.')
    elif liters_in_tank < 25:
        print(f'Fill your tank with {fuel}!')
else:
    print('Invalid fuel!')

File: JS Basics/02 Conditional Statements - More Exercises/08. Fuel Tank.js
function tank(fuel, liters) {
    let result = String(fuel);
    if (fuel == "Diesel" | fuel == "Gasoline" | fuel == "Gas") {
        if (Number(liters) >= 25) {
            console.log(`You have enough ${result.toLowerCase()}.`);
        }
        else {
            console.log(`Fill your tank with ${result.toLowerCase()}!`);
        }
    }
    else {
        console.log("Invalid fuel!");
    }
}
tank("Gasoline", 25)